webapp-django/crashstats/crashstats/models.py

```diff
@@ -26,6 +26,7 @@
 import socorro.external.postgresql.gccrashes
 import socorro.external.postgresql.crontabber_state
 import socorro.external.postgresql.adi
+import socorro.external.postgresql.product_build_types
```

Comment:
Maybe at some point we'll want to import the whole ``socorro.external.postgresql``, don't you think?